README.md

```diff
@@ -32,30 +32,55 @@ yarn add @hubspot/calling-extensions-sdk
 We have installed the SDK on a demo call widget as an example.
 
 ### Installation
-1. Download the demo call widget by cloning/forking this repo or by [downloading the ZIP](https://github.com/HubSpot/calling-extensions-sdk/archive/refs/heads/master.zip).
-2. In your terminal, locate the root directory of the project, then navigate to the demo directory by running:
+1. Clone or fork this repository to download the Demo Call Widget.
```

Comment:
I added the ZIP instructions from Deepthi's walkthrough since she didn't have a public GH account and wasn't looking to have one long-term.

Reply:
My thought process was to simplify the instructions since we can assume that our users are developers. However, we can include these instructions in our internal walkthroughs to help non-developers understand the steps.

Reply:
One of our goals is to expose the demo app to external non-developers so they can understand how to integrate their calling app into HubSpot without installation. Right now, we can't safely assume that only developers are coming to our README because it is linked in the [HubSpot Developer Docs](https://developers.hubspot.com/docs/api/crm/extensions/calling-sdk), which in turn is linked to anyone asking about HubSpot Third Party Calling. I think that doc and this README overlap - which can be confusing.

I've presented this to our quad, and as a result, we will be meeting with the developer relations and knowledge base teams next week to figure out our documentation strategy going forward. I agree with you in theory that GitHub docs are geared for devs, so I'll def pitch that to them and share the outcome of next week's discussion with you!

Reply:
Gotcha, thanks for explaining the long-term goal! I understand that non-developers could be following these steps to install the demo widget but even with the ZIP instructions, the README would still be pretty confusing to follow if someone is completely new to using developer tools. A potential solution would be to provide a more user friendly guide, perhaps in the form of screenshots or a video, in the HubSpot Developer Docs.

I do think that the short term objective of making the instructions concise and easy to follow for our current user base and upcoming bug bash session is important, and we can follow up with any necessary feedback

Reply:
Gotcha, thanks for explaining the long-term goal! I understand that non-developers could be following these steps to install the demo widget but even with the ZIP instructions, the README would still be pretty confusing to someone new to dev tools. A potential solution would be to provide a more user friendly guide, perhaps in the form of screenshots or a video, in the HubSpot Developer Docs.

I do think that the short term objective of making the instructions concise and easy to follow for our current user base is important - also I've omitted the old ZIP link that references `master` since the demo app is on a temporary `project-demo-v1` branch right now. We should probably have this documentation ready for the upcoming bug bash session and follow up with any guidance from the developer relations and knowledge base teams.